mesh-workshop.bat

```diff
@@ -0,0 +1,26 @@
+@echo off
+
+set confpath=%1
+set nodename=%2
+set HOSTNAMES="bathurst bayview bloor broadview chester christie college coxwell davisville donlands dufferin dundas dupont eglinton ellesmere finch greenwood islington jane keele kennedy king kipling lawrence leslie midland museum osgoode ossington pape queen rosedale runnymede spadina summerhill union warden wilson woodbine yorkdale"
+
+rem Check required inputs
+IF [%confpath%] == [] GOTO VARMISSING
+IF [%confpath%] == [] GOTO VARMISSING
+rem Make sure the node name is valid
+if NOT exist custom\conf.d\%nodename%\30-mesh-workshop.tar.gz goto VARMISSING
```

Comment:
There is no way to check from `HOSTNAMES` list? No way...

Even if we validate this way, we should check only up to `custom\conf.d\%nodename%`, otherwise one day we change `30-mesh-workshop.tar.gz` this script will need update.